src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Patch_V1Compat.txt

```diff
@@ -0,0 +1,19 @@
+﻿#SKIPFILE: disable:PowerFxV1CompatibilityRules
+#SETUP: EnableExpressionChaining,MutationFunctionsTestSetup
+
+// field22 is missing
+>> Patch(
+     Table(
+       {field1:2, field2:{field21:"earth",field22:"venus"}},
+       {field1:3, field2:{field21:"moon",field22:"phobos"}}),
+     {field2:{field21:"moon"}},
+     {field1:7})
+{field1:7,field2:{field21:"moon",field22:"phobos"}}
```

Comment:
This is wrong - we shouldn't have this result (see #1383)

Reply:
As discussed, replaced with #SKIP and the issue number.